docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,111 @@
+# Create a Bootable Pop!_OS USB in Linux
+
+Use this guide to create Pop!_OS bootable USB media from Linux:
+
+- Install ISO flashing software.
+- Download the Pop!_OS.iso file.
+- Verify the ISO.
+- Flash the ISO to your USB storage device.
+
+---
+## I. Install Popsicle
+
+![Popsicle Icon](/images/create-bootable-usb-linux/popsicle-icon.png)
+
+Create a bootable USB drive for Pop!_OS using Popsicle. Popsicle is a Linux utility for flashing multiple USB devices in parallel, written in Rust. You can install Popsicle via flatpak or AppImage.
+### AppImage
+
+1. Download the Popsicle AppImage [from this repo.](https://github.com/pop-os/popsicle/releases/latest)
+
+2. Navigate to the downloaded AppImage. Right-click the appimage and select Properties.
+
+3. Select the `Permissions` tab and check the `Execute` box.
+
+![AppImage Execute Permission](/images/create-bootable-usb-linux/appimage-execute.png)
+
+4. Double click the Popsicle AppImage to launch it.
+### Flatpak
+
+Linux users with [Flatpak installed](https://flatpak.org/setup/) in their distribution can simply download and install the [Popsicle flatpak](https://flathub.org/apps/details/com.system76.Popsicle) from Flathub. 
+
+
+## II. Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](create-bootable-usb.md#standard-nvidia--ras-pi-4-isos) for information about each version.
+
+## III. Verify the Pop!_OS.iso File
+
+You should verify that your download matches the checksum before trying to install. This ensures that you've received the full, complete download and that it is not corrupted. Compare the hash values generated using the below commands to the values shown on the [Pop!_OS download page](https://pop.system76.com/). **Do not use the image below to verify hash values.**
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+
+When entering the appropriate checksum command below, ensure:
+
+- The file name in your command matches the downloaded ISO in your local directory. 
+- The examples below use the ~/Downloads folder. Make sure the appropriate explicit or relative path is used in your command.
+
+>**Note**: Popsicle also provides a feature for easily checking the SHA256 checksum of an ISO file (Step 3). Skip this section if you prefer a GUI method.
```

Comment:
Why are we spending time explaining the CLI here if we can do this through the GUI that we instructed the user to download anyway? I would be in favor of removing this entire section and move the relevant parts into the next section.